databricks/koalas/series.py

```diff
@@ -1395,6 +1396,52 @@ def to_latex(
 
     to_latex.__doc__ = DataFrame.to_latex.__doc__
 
+    def to_markdown(self, buf=None, mode=None):
```

Comment:
Can we move this implementation to `generic.py`?

Reply:
Sure. Thanks for the comment!

Reply:
Moved !